### PR TITLE
restructure modules to match the services

### PR DESCRIPTION
**Pull request by caravinci**
As per specs, `A3` (short for Arranger 3.0, the rewrite project) will be shaped as follows:

- Server (Contains `schema`, `mapping-utils`, `middleware` and `admin`)
- Admin-UI (foundation for what will be a standalone service. UI only for now)
- Components (duplicates some aspects of `mapping-utils` and `middleware`)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,7 +66,8 @@ Now you should be able to start the following processes from the project's root
 
 #### - Dockerized Setup
 
-A bit more friendly "quickstart", if you just want to get things started
+A bit more friendly "quickstart", if you just want to get things started.
+Note: it may take a while on the first run, as it builds the images, etc.
```
Comment (joneubank) [inline README.md:70]: 👍

File: modules/components/src/AdvancedFacetView/utils.js [hunk 2]
```diff
@@ -1,9 +1,24 @@
...
+      path: currentPath,
+      id: `${key}`,
+      ...(fieldProps.properties
+        ? {
+            children: elasticMappingToDisplayTreeData(fieldProps.properties, currentPath),
+          }
+        : {}),
+    };
+  });
+};
```
Comment (joneubank) [inline modules/components/src/AdvancedFacetView/utils.js:21]: 👍  simple solution

File: package.json [hunk 3]
```diff
@@ -6,13 +6,13 @@
...
-    "bootstrap": "lerna bootstrap --hoist",
+    "bootstrap": "lerna bootstrap",
     "lerna": "lerna",
     "link": "npm-link-shared modules",
     "publish::ci": "lerna publish from-git --yes --force-publish=*",
     "test": "lerna run test --stream",
-    "server": "cd modules/server && npm run watch",
-    "run-prod-server": "cd modules/server && npm run run-prod",
+    "server": "cd modules/server && npm run start",
+    "server:prod": "cd modules/server && npm run start:prod",
```
Comment (joneubank) [inline package.json:15]: much better script naming.

**Review by joneubank — APPROVED** [review]

File: modules/server/src/projects/getFields.js [hunk 4]
```diff
@@ -1,8 +1,7 @@
-import { extendFields, extendMapping, loadExtendedFields } from '@arranger/mapping-utils';
-
+import { extendFields, extendMapping, loadExtendedFields } from '../mapping';
+import { fetchMapping } from '../mapping';
```
Comment (caravinci) [inline modules/server/src/projects/getFields.js:2]:
```suggestion
import { extendFields, extendMapping, fetchMapping, loadExtendedFields } from '../mapping';
```

**Review by caravinci — COMMENTED** [review]
A few notes to assist combing through these changes (apart from the files renaming/relocation)
e.g. Documentation to inform future hindsight

**Pull request merged**